Makefile

```diff
@@ -1,22 +1,23 @@
 PREFIX ?= /usr
+DESTDIR ?=
 BINDIR ?= $(PREFIX)/bin
 LIBDIR ?= $(PREFIX)/share/hilbish
 
 build:
 	@go build
 
-install:
-	@install -v -d "$(BINDIR)/" && install -m 0755 -v hilbish "$(BINDIR)/hilbish"
-	@mkdir -p "$(LIBDIR)"
-	@cp libs preload.lua .hilbishrc.lua "$(LIBDIR)" -r
-	@echo /usr/bin/hilbish >> /etc/shells
+install: build
```

Comment:
whyd u add back build >:|

Reply:
some people just do make install, and if they do it wont install cuz its not built :/ this prob is only on ur system lol

Reply:
`sudo make` is still available tho